packages/at_commons/CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+## 4.0.4
+- feat: Enhance enroll:list to enable filtering of enrollments based on their status
```

Comment:
Enhance enroll:list to enable filtering  based on enrollment status

Reply:
change made as part of [feddc9e](https://github.com/atsign-foundation/at_libraries/pull/530/commits/feddc9e80d5327223fa0a5fed5f6ff9a3173fe56)